examples/dam.jl

```diff
@@ -62,7 +62,7 @@ end
 in advance."""
 function solve_determinist_problem()
     println(alea_year)
-    m = Model(solver=CplexSolver(CPX_PARAM_SIMDISPLAY=0))
+    m = Model(solver=SOLVER)
 
     @defVar(m,  0           <= x[1:(TF+1)]  <= 100)
```

Comment:
Seems good to me. Would just prefer to see 7 and 100 to be constants defined at the beginning of the file.